src/pages/home/Home.js

```diff
@@ -1,5 +1,15 @@
 import React from 'react';
+import { ModalTrigger } from '@nomios/web-uikit';
 
-const Home = () => <div>This is the home page!</div>;
+import NewIdentityFlow from '../../modals/new-identity-flow';
+
+const Home = () => (
+    <div>
+        <h4>This is the home page!</h4>
+        <ModalTrigger modal={ <NewIdentityFlow /> }>
```

Comment:
Instead of doing this here, can you integrate this in the sidebar? We already have a + button there.

Reply:
Sure!